frontend/src/js/controllers/authCtrl.js

```diff
@@ -194,6 +194,28 @@
             utilities.sendRequest(parameters, "no-header");
         }
 
+        vm.verifyEmail = function() {
+            vm.startLoader("Verifying Your Email");
+            var parameters = {};
+            parameters.url = 'auth/registration/account-confirm-email/' + $state.params.email_conf_key + '/';
+            parameters.method = 'GET';
+            parameters.callback = {
+                onSuccess: function(response) {
+                    vm.email_verify_msg = "Your email has been verified successfully";
+                    vm.stopLoader();
+                    $timeout(function() {
+                        $state.go("auth.login");
```

Comment:
lets not redirect the user directly to the login page. Lets give a button on that view from where the user can go to Login View.